build.gradle

```diff
@@ -2861,7 +2862,7 @@ project(":media") {
                     args("JAVA_HOME=${JDK_HOME}", "GENERATED_HEADERS_DIR=${generatedHeadersDir}",
                          "OUTPUT_DIR=${nativeOutputDir}", "BUILD_TYPE=${buildType}", "BASE_NAME=jfxmedia",
                          "COMPILE_PARFAIT=${compileParfait}",
-                         IS_64 ? "ARCH=x64" : "ARCH=x32",
+                         IS_64 ? IS_AARCH64 ? "ARCH=aarch64" : "ARCH=x64" : "ARCH=x32",
```

Comment:
Does `ARCH=aarch64` get handled correctly on Mac and Windows?

Reply:
We don't have media/web on Windows-aarch64 yet.
On mac, it is handled differently: if a TARGET_ARCH is specified, the mac.gradle will add -target TARGET_ARCH-apple-macos-11 to the CC parameter which is passed from build.gradle to the media build scripts.

I realise we are not consequent here: we specify the ARCH to the makefile, but we also provide the CC and LINKER properties which can contain the architecture as well.

Reply:
OK, we can do any cleanup in the follow-on issue you plan to file to make our handling of aarch64 more consistent.